examples/c_dgfip/README.md

```diff
@@ -0,0 +1,88 @@
+## Using Mlang with C
+
+Mlang is able to compile the official source code of the French income tax
+computation to C. This means that you can embark the computation as a
+regular C source code file in your application if you need it. This file
+is a guide about how to do it.
+
+### Generating the C file
+
+You have to invoke Python for that, with something like:
+
+```
+mlang --display_time --debug \ # Prints debug information
+    --mpp_file=path/to/mlang/<year>.mpp \  # <year> is the year when the income was received
+    --mpp_function=compute_double_liquidation_pvro \ # this function is the one computing the taxes
+    --optimize \ # optimize the output, otherwise the generated file is very large
+	--backend c --output <name>.py \ # outputs a Python file
+	--function_spec <spec_file>.m_spec \ # configuration (see above)
+	<M source files from ir-calcul>
+```
+
+### Using the generated C file
+
+The main function provided by the generated C file provides one function,
+`m_extracted`. This functions takes a struct as its argument, and returns another
+struct.
+
+he input struct has its own type, `m_input`, whose fields are the different
+input values of the income tax computation. Each value has type `m_value`, which
+is a `double` wrapped with a boolean flag to indicate if the value is undefined.
```

Comment:
I don't think this doc is up to date with what you plan to do with the DGFiP backend, I would prefer you remove the text and put a placeholder until you write specific doc for it.